### PR TITLE
Remove.dust

### PR DESCRIPTION
**Pull request by grawk**
- remove formatDust from the API
- mention a custom implementation for formatDust in README
- add the custom formatter into the unit tests

### CODE REVIEW

**Comment by grawk** [comment]
@aredridel this feels like the right way forward, as we begin moving our examples and other documentation away from dust as a specific solution. please let me know what you think. i'd like to get `1.0.0` published hopefully soon.

File: README.md [hunk 1]
```diff
@@ -78,6 +74,32 @@ bundle.get('bundle': {
...
+        }
+
+        dust.render(this.cache[pattern], model, renderCb);
+    };
+    return bundle;
+};
+
+bundler.get({bundle: 'errors/server', locality: 'en-US'}, function bundaloReturn(err, bundle) {
+    bundle = dustFormatter(bundle);
+    console.log("what'd we get from bundalo.get?", data, err);
```
Comment (shaunwarman) [inline README.md]: Just an example log?

File: package.json [hunk 2]
```diff
@@ -32,18 +32,18 @@
   },
   "homepage": "https://github.com/krakenjs/bundalo",
   "devDependencies": {
+    "dustjs-linkedin": "~2.7.2",
```
Comment (shaunwarman) [inline package.json:35]: Want to stay explicitly at `2.7.2` with `~`?

**Comment by shaunwarman** [comment]
👍

File: package.json (hunk 2 above)
Comment (grawk) [inline package.json:35]: Unfortunately the dust module has been known to release breaking changes in minor releases.

Comment (shaunwarman) [inline package.json:35]: Ya this sounds familiar.

**Comment by gabrielcsapo** [comment]
👍

**Pull request merged**